src/main/java/ch/njol/skript/lang/function/FunctionReference.java

```diff
@@ -250,7 +250,8 @@ public boolean resetReturnValue() {
 			function = (Function<? extends T>) Functions.getFunction(functionName);
 		
 		if (function == null) { // It might be impossible to resolve functions in some cases!
-			Skript.error("Invalid function call to a function that does not exist yet. Be careful when using functions in 'script load' events!");
+			Skript.error("Invalid function call to a function that does not exist yet, named '" + functionName
+				+ "'. Be careful when using functions in 'script load' events!");
```

Comment:
It may be good to change the error to something else, as it doesn't make much sense right now, like:
```suggestion
			Skript.error("Couldn't resolve call for '" + functionName + 
			"'. Be careful when using functions in 'script load' events!");
```